lib/galaxy/webapps/galaxy/api/tools.py

```diff
@@ -360,6 +360,7 @@ def _search(self, q):
         :return type: dict
         """
         tool_name_boost = self.app.config.get('tool_name_boost', 9)
+        tool_id_boost = self.app.config.get('tool_id_boost', 9)
```

Comment:
We should add the config option with its default value to the sample file.

Reply:
@juleengraham could you do this please ? I think this is ready to be merged otherwise.